### PR TITLE
Github5420

### LINKED ISSUE #5420: Multi-coloured highlights go wrong with small fonts
**Describe the bug**
image

**To Reproduce**
In test1.cpp, testGithub2931_1, set the draw option to a fixed font size of 10.

**Expected behavior**
The bond highlights would be parallel to the bonds.

**Screenshots**

**Configuration (please complete the following information):**
2022.03.3, built from source on macOS Monterey and Apple silicon

**Additional context**
The ellipses are smaller than the width of the highlight, so the lines miss and the fallback position puts them back to the centre of the drawing.
I'll fix it tomorrow.

### PR DESCRIPTION
**Pull request by DavidACosgrove**
#### Reference Issue
Fixes #5420

#### What does this implement/fix? Explain your changes.
The bond highlights were sometimes missing the atom highlights and the fallback position was incorrect.
Fallback now correct.
The miss was occurring in the test case because the atom highlight wasn't centred on the atom, it was centred on the centre of the atom label which for a multiple character label was off to one side.  It now uses the label centre.  Only really an issue with small fonts.

#### Any other comments?
Using the label centre now means that the bond highlights aren't always parallel with the bond, but I think that's ok.  The alternative would be to make them narrower, so they still went parallel to the bond but hit the atom highlight very much off-centre.  That could be done, but would look differently messy, I think.  It's an edge case.

### CODE REVIEW

**Review by greglandrum — CHANGES_REQUESTED** [review]
Code looks fine. Only remaining suggestion is to re-enable deleting the test output.

File: Code/GraphMol/MolDraw2D/test1.cpp [hunk 1]
```diff
@@ -357,7 +359,7 @@ using namespace RDKit;
 // if the generated SVG hashes to the value we're expecting, delete
 // the file.  That way, only the files that need inspection will be
 // left at the end of the run.
-static const bool DELETE_WITH_GOOD_HASH = true;
+static const bool DELETE_WITH_GOOD_HASH = false;
```
[the remark this answers is not in the document]
Comment (DavidACosgrove) [reply Code/GraphMol/MolDraw2D/test1.cpp:362]: Oops, sorry.
It's possible that some of the hash codes need updating, as well, but given the ACS1996 PR is also going to break pretty much all of them, it doesn't seem worth fixing them at the moment.

**Review by greglandrum — APPROVED** [review]
LGTM
